types/coin.go

```diff
@@ -297,12 +314,26 @@ func (coins Coins) SafeSub(coinsB Coins) (Coins, bool) {
 // IsAllGT returns true if for every denom in coins, the denom is present at a
 // greater amount in coinsB.
```

Comment:
This seems to contradict the function's behaviour, which checks if the denom is present at a greater amount in `coinsA`.